BUILD.gn

```diff
@@ -143,6 +143,7 @@ deno_cargo_info = exec_script("build_extra/rust/get_cargo_info.py",
 rust_executable("deno") {
   source_root = "src/main.rs"
   extern = main_extern
+  features = [ "use-snapshot-init" ]
```

Comment:
I'd rather snapshots be opt-out rather than opt-in. Is that doable?

Reply:
That is change the arm feature to be something like "no-snapshots"

Reply:
I was thinking that might be a good idea since x86_64 native builds are still the primary goal. I came up with a better solution that should add `no-snapshot-init` when target_arch != host_arch.

Reply:
Cool that's better